### DISCUSSION TITLE
无法通过钉钉登录

### DISCUSSION THREAD

**Discussion by lic17** [hhyo/Archery#2495] [Q&A] [answered]
配置钉钉登录后，点击钉钉登录后，总会重新跳转到登录页面，无法登录，也没有任何报错信息存在。setting配置如下：
![图片](https://github.com/hhyo/Archery/assets/18326247/3542e142-72fd-4230-859e-bd7ce4185ea5)

**Comment by LeoQuote** [comment, marked as answer]
用master分支的代码或者镜像重试

**Reply by lic17** [reply]
我用master也不行呢，浏览器跳转登录时会像下图这样访问这几个地址很奇怪，然后还是跟原来一样登录不了
<img width="1426" alt="图片" src="https://github.com/hhyo/Archery/assets/18326247/d0062efe-681c-45f1-8ad4-2480fbcf8268">

**Reply by lic17** [reply]
已经解决了，用户不属于对应的企业造成的，主要是没有报错和日志，不太好排查
